types/deno/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "Node16",
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "types": [],
+        "lib": [
+            "DOM",
```

Comment:
Ah, because I had it here. I'll look into removing this.

Reply:
Actually, removing this requires a pretty significant change. Might be better to just require people to use "DOM" for now, then we can look into creating all the conditional types later so it works well with or without dom types.